build.sbt

```diff
@@ -14,7 +14,13 @@ lazy val commonSettings = Seq(
                                                  |See the NOTICE file distributed with this work for
                                                  |additional information regarding copyright ownership.
                                                  |""".stripMargin)),
-  scalaModuleMimaPreviousVersion := Some("2.1.1"),
+  scalaModuleMimaPreviousVersion := {
+    // We need to create a release version first to use MiMa
+    if(sys.env.get("SCALAJS_VERSION").exists(_.startsWith("1.0.0-M")))
```

Comment:
Should be `1.0.0-`, no? (without the `M`)

Reply:
I was thinking that as well if we are changing to `RC1` ...

Reply:
Also, it should be `forall` rather than `exists` because we still want to enable MiMa when building JVM artifacts.

Reply:
Uh, I’m sorry but my reasoning was wrong, using `exists` as you initially did was the correct thing to do…